app/models/visited_sight.rb

```diff
@@ -0,0 +1,8 @@
+class VisitedSight < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :sight
+
+  validates :user_id, presence: true
+  validates :sight_id, presence: true
+  validates_uniqueness_of :sight_id, scope: :user_id
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.